tonic_validate/metrics/answer_consistency_metric.py

```diff
@@ -22,7 +24,8 @@ def __init__(self):
         Metric that checks whether the LLM answer contains information that does not come from the context.
         Returns a float between 0 and 1, where 1 is completely consistent and 0 is completely inconsistent.
         """
-        pass
+        self.prompt = f'-------------------\n{main_points_prompt()}\n-------------------\n'
```

Comment:
1. Can we move the prompt out of the constructor so it can be called without instantiating the class? Could make it a class attribute by just putting `prompt = "whatever"` at the top of the class.
2. In metric.py, we should also set the class attribute there too like so `prompt: Optional[str] = None`. That way if llama index calls prompt for a non llm metric it won't fail
3. On this prompt specifically, we could combine it just into one string. with multiple lines like so
```
prompt = (
            "-------------------\n"
            f"{main_points_prompt()}\n"
            "-------------------\n"
            f"{statement_derived_from_context_prompt(statement="EXAMPLE STATEMENT", context_list=[])}\n"
            "-------------------\n"
        )```

Reply:
Did this and fixed the rest of the classes